express/code/scripts/scripts.js

```diff
@@ -227,9 +227,45 @@ if (new URLSearchParams(window.location.search).get('lingo')) {
 document.body.dataset.device = navigator.userAgent.includes('Mobile') ? 'mobile' : 'desktop';
 preDecorateSections(document);
 // LCP image decoration
+const eagerLoad = (img) => {
+  img?.setAttribute('loading', 'eager');
+  img?.setAttribute('fetchpriority', 'high');
+};
+
 (function decorateLCPImage() {
-  const lcpImg = document.querySelector('img');
-  lcpImg?.removeAttribute('loading');
+  const firstSection = document.querySelector('body > main > div:nth-child(1)');
+  if (!firstSection) return;
+
+  // Get all images in the first section
+  const images = firstSection.querySelectorAll('img');
+  if (images.length > 0) {
+    images.forEach(eagerLoad);
```

Comment:
isn't this sort of just doing duplicate work milo is already doing? https://github.com/adobecom/milo/blob/stage/libs/scripts/scripts.js#L91

Reply:
seems Milo is only targeting blocks containing 'marquee'. We are extending it to the first block.

Reply:
got it... so Milo's behavior is: first block is marquee => eager load all images in it. first block is not marquee, only eager load the first image. Why do we want to deviate from this behavior? Is it common that our LCPs are non-first images in the first section?
If you can show us proper before vs after, that would make the extra code more convincing. Like I mentioned in the other PR, you'll want to turn off martech for things to be repeatable. Otherwise in general, I would tend to be extra cautious every time I see new code or complexity being added to improve performance. I am pleased more easily when seeing performance PR touching scripts.js that "removes" or "optimizes" code rather than adding, which ships more bytes in the critical path.